### PR TITLE
This fixes the crab

### LINKED ISSUE #10427: Everything about MEGAMADCRAB is hilariously broken
For those who don't know, using wirecutters on a crab has a 50% chance to instantly kill it, and a 50% chance to turn it into a MEGAMADCRAB, a very strong and angry crab. It is broken in just about every way you can imagine.
1. The living and dead sprites for MEGAMADCRAB are exactly the same, and neither is a crab. In fact, they are both naked white human males.
2. MEGAMADCRABs are not actually hostile.
3. You still have a 50% chance of killing them instantly with wirecutters, making them a negligible threat even if they did actually attack you.
4. If you examine them, your client freezes for a moment and the examine icon is horribly distorted, just like #10053 (@PJB3005)
5. Despite there being several different crab appearances, all of them turn into the same sprite. Which would be bad even if that sprite were a crab.

As an added bonus, if this DID work properly, wirecuttering Norris repeatedly would turn him into an unstoppable murder machine, because he revives to full health a few seconds after dying.

### PR DESCRIPTION
**Pull request by ghost**
Some absolute madman called Pomf uncommented something called MEGA MAD CRAB, which is so weirdly coded it spawned a naked bald dude and crashed the server eventually.

This should be slightly more sensible.

Fixes #10427

### CODE REVIEW

**Comment by Exxion** [comment]
> Some absolute madman

It was Pomf, at the end of 2014

**Comment by ghost** [comment]
The hell failed? It was a fresh pull

**Comment by Exxion** [comment]
`code/modules/mob/living/simple_animal/friendly/crab.dm:60:error: /mob/living/simple_animal/hostile/crab: undefined type path`

You didn't commit the DME change

**Comment by ghost** [comment]
Woops, hopefully this'll fix it

File: code/modules/mob/living/simple_animal/hostile/crab.dm [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	speak_emote = list("clicks")
+	emote_hear = list("clicks with fury", "clicks angrily")
+	emote_see = list("clacks")
+	speak_chance = 1
+	turns_per_move = 15//Gotta go fast
+	maxHealth = 300//So they don't die as quickly
+	health = 300
+
+
+	melee_damage_lower = 10
```
Comment (Iamgoofball) [inline code/modules/mob/living/simple_animal/hostile/crab.dm:20]: attacks from the weak point

File: code/modules/mob/living/simple_animal/hostile/crab.dm [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	melee_damage_upper = 15
```
Comment (Iamgoofball) [inline code/modules/mob/living/simple_animal/hostile/crab.dm:21]: for massive damage

**Comment by AutismSpider** [comment]
if going be massive damage should be 20 for upper

File: code/modules/mob/living/simple_animal/hostile/crab.dm (hunk 2 above)
Comment (Exxion) [inline code/modules/mob/living/simple_animal/hostile/crab.dm:21]: There are only two crabs on the station, and you have a 50% chance of simply killing one of them if you try to MEGAMADCRAB it. Plus goliaths deal 25 damage anyway, and you can get way more of those if you want to kill the crew.

**Comment by AutismSpider** [comment]
three crabs on bus

**Comment by Imagebard** [comment]
50% chance of crab death, 50% chance of megamadcrab, 150% chance of memes.

**Comment by SkowronX** [comment]
fuckin byond

**Comment by Kurfursten** [comment]
I feel as though if you attack MEGAMADCRAB with a knife, it should break and the crab should rip out your eyes.

Never bring a knife to a crab battle.

**Comment by Shadowmech88** [comment]
Better yet, have the crab take the knife and double the crab's damage.

**Comment by AutismSpider** [comment]
https://www.youtube.com/watch?v=t4enrU2BMpI

make the AI pick up any near by knifes/swords and use it?

**Comment by Kurfursten** [comment]
https://www.youtube.com/watch?v=8mHKHKR8x6A

But those ideas are fun too

**Comment by ghost** [comment]
Who changed my PR name?

**Comment by Kurfursten** [comment]
clusterfack 8 hours ago

You can always check the log of PR name changes by scrolling up

File: code/modules/mob/living/simple_animal/hostile/crab.dm (hunk 2 above)
Comment (ghost) [inline code/modules/mob/living/simple_animal/hostile/crab.dm:21]: Keeping it fairly low as all it takes is 2 snips and you can no longer run faster than it, kinda ensuring death by crab

**Comment by ghost** [comment]
So, what's there to be done?

A buff is all I'm seeing requested.

**Comment by Exxion** [comment]
If it's possible to make them attack ABSOLUTELY EVERYTHING ALIVE, that would be cool, but if it's too much trouble, not a big deal.

**Comment by ghost** [comment]
They have no faction, they hate everything except MoMMIs because MoMMIs have been snowflaked to be loved by all.

**Comment by Exxion** [comment]
:+1:
That even kind of makes sense because MoMMIs are robo-crabs

**Comment by FudgePucker** [comment]
Why would the crab attack its brother?

**Comment by FudgePucker** [comment]
Also blue eye mommis can't fight back against simple mobs. So if carp/ other mobs attacked them they would have to accept their fate instead of just weldering it to death

**Comment by ComicIronic** [comment]
This needs logging for the griff potential.

**Comment by ghost** [comment]
One set of admin log attempts coming up

**Comment by ComicIronic** [comment]
I suggest you look at (and use) add_logs()

**Comment by ghost** [comment]
Thanks, was about to ask what I should be looking for

**Comment by ghost** [comment]
Am I doing add_logs or add_gamelogs?

I'm assuming I'll also do an admin jumpto sorta thing so they can watch the fireworks?

**Comment by ghost** [comment]
creb

Am I doing this right? (Ignore the double fullstop, fixed that already)

**Comment by NigglyWiggly** [comment]
I feel like you might have have messed something up a little bit

**Comment by ghost** [comment]
What exactly?

File: code/modules/mob/living/simple_animal/friendly/crab.dm [hunk 3]
```diff
@@ -43,39 +43,22 @@
 
 //LOOK AT THIS - ..()??
 /mob/living/simple_animal/crab/attackby(var/obj/item/O as obj, var/mob/user as mob)
+	if(stat == DEAD)
+		return
```
Comment (unid15) [inline code/modules/mob/living/simple_animal/friendly/crab.dm]: This might break butchering, do return ..() instead

**Comment by AutismSpider** [comment]
All most ready.Could use better attack sounds but what ever.

mad crab

**Comment by ghost** [comment]
Unless anyone else has any qualms, should be done.

**Comment by ghost** [comment]
It's time for snib

**Comment by ComicIronic** [comment]
SCHNIBBEN SCHNABBEN

**Pull request merged**